charts/spire/README.md

```diff
@@ -47,6 +47,21 @@ spec:
         - --service-account-signing-key-file=/run/config/pki/sa.key
 ```
 
+## Deployment - Non Production
```

Comment:
Reorder this to lead with the approach we want people to take first.

This means that we want the "Production" deployment first, because that's our mission to deliver, and we want people to do it this way.

Which brings the question, do we even want to advertise a different way?  Do we need to?  Savvy developers of Helm will know what to do if we don't tell them, and the dangers that come with their own choices.

We should be very careful advertising how to do something we don't really want to support.